projects/platform-terminal/src/lib/adapters/line.ts

```diff
@@ -0,0 +1,6 @@
+import { Widgets } from 'blessed';
```

Comment:
I'm pretty sure we don't need to create an adapter just 'for the future'. If we don't need an adapter - we don't create an adapter.